bin/node/bench/src/import.rs

```diff
@@ -127,15 +127,10 @@ impl core::Benchmark for ImportBenchmark {
 		context.import_block(self.block.clone());
 		let elapsed = start.elapsed();
 
-		let hash = context
-			.client
-			.expect_block_hash_from_id(&BlockId::number(1))
-			.expect("Block 1 was imported; qed");
-
 		// Sanity checks.
 		context
 			.client
-			.state_at(hash)
+			.state_at(self.block.header.hash())
```

Comment:
not related to other changes, removed excessive call to `expect_block_hash_from_id`.